.github/workflows/ci.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           scarb --manifest-path examples/spawn-and-move/Scarb.toml fmt --check
           scarb --manifest-path examples/simple/Scarb.toml fmt --check
           scarb --manifest-path crates/dojo/core/Scarb.toml fmt --check
-          scarb --manifest-path crates/dojo/core-cairo-test/Scarb.toml fmt --check
+          scarb --manifest-path crates/dojo/core-foundry-test/Scarb.toml fmt --check
```

Comment:
Wondering if we should keep both during a time? Since everybody may not migrate to foundry (at least for now?).

Reply:
But proc macros are disabled with the cairo test runner, so if we keep `core-cairo-test`, I think we also have to keep the Dojo plugin 😕 I don't know if it's possible to mix both plugin and proc macros ?